django/http/response.py

```diff
@@ -299,10 +299,15 @@ def __init__(self, content=b'', *args, **kwargs):
         self.content = content
 
     def __repr__(self):
-        return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+        if 'Content-Type' in self:
+            return '<%(cls)s status_code=%(status_code)d, "%(content_type)s">' % {
+                'cls': self.__class__.__name__,
+                'status_code': self.status_code,
+                'content_type': self['Content-Type'],
+            }
+        return '<%(cls)s status_code=%(status_code)d>' % {
```

Comment:
If it has some readability benefits, it could be done in a separate PR. This looks okay for now.